FlashDevelop/Bin/Debug/Projects/310 Haxe - Flash Project/Project.hxproj

```diff
@@ -8,7 +8,7 @@
     <movie fps="30" />
     <movie width="800" />
     <movie height="600" />
-    <movie version="17" />
+    <movie version="18" />
```

Comment:
We need to check what FP version is supported by the Haxe compiler - Haxe 3.1.3 didn't even accept FP17.